core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -1044,7 +1045,7 @@ public static function addPageIcon($row, $label, ?DataContainer $dc=null, $image
 		}
 
 		// Add the breadcrumb link if you have access to that page
-		if (!$isVisibleRootTrailPage)
+		if ($objUser->hasAccess($row['id'], 'pagemounts'))
```

Comment:
Currently, you cannot click on a root trail page in the page tree to create a breadcrumbs from there. However, you can click on the same page if the breadcrumb has previously been set to a nested subpage of it. This change unifies that, I think it makes sense to allow the breadcrumb for any nested level, since I can see this anyways.